spec/syntax.md

```diff
@@ -370,19 +370,15 @@ option = name [s] "=" [s] (literal / nmtoken / variable)
 Examples:
 
 ```
-(1.23)
+|1.23|
 ```
 
 ```
-(1.23) :number maxFractionDigits=1
+|1.23| :number maxFractionDigits=1
 ```
 
 ```
-(1970-01-01T13:37:00.000Z) :datetime weekday=long
-```
-
-```
-(Thu Jan 01 1970 14:37:00 GMT+0100 \(CET\)) :datetime weekday=long
+|Thu Jan 01 1970 14:37:00 GMT+0100 (CET)| :datetime weekday=long
```

Comment:
Ew. Keep the ISO8601 format for the literal.

Reply:
Keep both or keep iso8601 only?

Reply:
I would only use the ISO8601 format. It doesn't make sense to have the RFC format and associate it with a formatter.

If we wanted to have a timezone, the emerging extension to 8601 for Temporals is to include the IANA/Olson name, either in the form of a zoned offset date time:

>2007-01-01T01:00:00.000+01:00[America/Chicago]

Or a zoned instant:

> 2007-01-01T01:00:00.000Z[America/Chicago]

See [here](https://w3c.github.io/timezone/#serializations) for more examples.

Reply:
Agreed.
```suggestion
|1970-01-01T14:37:00+01:00[Europe/Berlin]| :datetime weekday=long
```